src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -70,48 +88,22 @@ internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSynt
                     if (this.CanAddImport(node, cancellationToken))
                     {
                         var semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
-
-                        var allSymbolReferences = new List<SymbolReference>();
-
-                        var finder = new SymbolReferenceFinder(this, document, semanticModel, diagnostic, node, cancellationToken);
-
-                        // Caches so we don't produce the same data multiple times while searching 
-                        // all over the solution.
-                        var projectToAssembly = new ConcurrentDictionary<Project, AsyncLazy<IAssemblySymbol>>(concurrencyLevel: 2, capacity: project.Solution.ProjectIds.Count);
-                        var referenceToCompilation = new ConcurrentDictionary<PortableExecutableReference, Compilation>(concurrencyLevel: 2, capacity: project.Solution.Projects.Sum(p => p.MetadataReferences.Count));
-
-                        // Look for exact matches first:
-                        await FindResults(projectToAssembly, referenceToCompilation, project, allSymbolReferences, finder, exact: true, cancellationToken: cancellationToken).ConfigureAwait(false);
-                        if (allSymbolReferences.Count == 0)
-                        {
-                            // No exact matches found.  Fall back to fuzzy searching.
-                            // Only bother doing this for host workspaces.  We don't want this for 
-                            // things like the Interactive workspace as this will cause us to 
-                            // create expensive bktrees which we won't even be able to save for 
-                            // future use.
-                            if (IsHostOrTestWorkspace(project))
-                            {
-                                await FindResults(projectToAssembly, referenceToCompilation, project, allSymbolReferences, finder, exact: false, cancellationToken: cancellationToken).ConfigureAwait(false);
-                            }
-                        }
+                        var allSymbolReferences = await FindResultsAsync(document, semanticModel, diagnostic, node, cancellationToken).ConfigureAwait(false);
 
                         // Nothing found at all. No need to proceed.
-                        if (allSymbolReferences.Count == 0)
+                        if (allSymbolReferences == null || allSymbolReferences.Count == 0)
```

Comment:
does that actually work?  if the list is null, won't .Count return a Nullable int.  And a nullable int without a value won't be equal to 0.